src/cpu-template-helper/src/utils.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::ffi::OsString;
+use std::path::{Path, PathBuf};
+
+pub fn add_suffix(path: &Path, suffix: &str) -> PathBuf {
+    // Extract the part of the filename before the extension.
+    let mut new_file_name = OsString::from(path.file_stem().unwrap());
+
+    // Push the suffix and the extension.
+    new_file_name.push(suffix);
+    if let Some(ext) = path.extension() {
+        new_file_name.push(".");
+        new_file_name.push(ext);
+    }
+
+    // Swap the file name.
+    path.with_file_name(new_file_name)
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    static SUFFIX: &str = "_suffix";
```

Comment:
To remove duplication, this value can also be used when defining clap:
```
        /// Suffix of output files. To overwrite input files, specify an empty string ''.
        #[arg(short, long, default_value = SUFFIX)]
        suffix: String,
```
This can also be a constant and I would prefer renaming it:
```suggest
    const DEFAULT_SUFFIX: &str = "_suffix";
```

Reply:
> This can also be a constant and I would prefer renaming it:

I agree that it can be a constant, but I'm not sure if renaming to `DEFAULT_SUFFIX` is beneficial, because the `SUFFIX` is only used in tests and there is no custom suffix.

> To remove duplication, this value can also be used when defining clap:

I may miss something, but I'm not sure which part is duplicate. the default suffix for the strip command (`_stripped`) is only used here and `utils.rs` is a separated module from `main.rs`.